Api/ride.py

```diff
@@ -49,35 +45,30 @@ def get(self,location):
         for offer in ride_Offers:
             if offer["location"] == location:
                 list_of_offers.append(offer)
-            return {"list of offers":list_of_offers},200
+        return {"list of offers":list_of_offers},200
     
 class DriverRideOffer(Resource):  
     def post(self):
         #driver post ride offer data
         postoffer = request.get_json()
-        # validate it
-        data, errors = rideschema.load(postoffer)
+        data,errors = rideschema.load(postoffer)
         if errors:
-            return make_response(jsonify(errors), 400)
-           
+            return{'error':errors}
         #create an instance of class RideOffer
 
         new_offer = DriverOffer(postoffer.get("location"),
-                                postoffer.get("destination"),
-                                postoffer.get("driver_details")
+                                postoffer.get("destination")
                                 )
-        DT=json.dumps(new_offer.departure)         
-
-        # save the new_offer to ride_offers[]
+        DT=json.dumps(new_offer.departure)        
+                # save the new_offer to ride_offers[]
 
         ride_Offers.append({"id":new_offer.ride_id,
                             "location":new_offer.location,
                             "destination":new_offer.destination,
                             "departure":DT,
-                            "driver details":new_offer.driver_details
+                            "driver details":new_offer.driver_detail
         })
-
-
+        # import pdb;pdb.set_trace()
```

Comment:
It's good to remove debuggers from your code and by doing this you will avoid performance and security issues as well.